lgtm.yml

```diff
@@ -0,0 +1,12 @@
+queries:
+
+  # Suppress some LGTM warnings.
+
+  # A module is imported with the "import" and "import from" statements.
+  # https://lgtm.com/rules/1818040193/
+  - exclude: py/import-and-import-from
```

Comment:
does this work? https://github.com/crate/crate-python/pull/391#issuecomment-1143438689

Reply:
No it doesn't ;[. At least, it does not work while not being merged. I hope that the additional rule will be recognized after the merge. However, it is weird because the other rule apparently seems to work.